src/pdf.rs

```diff
@@ -0,0 +1,79 @@
+use std::{env, path::PathBuf};
+
+use image::DynamicImage;
+
+use pdfium_render::prelude::*;
+
+pub enum PDFQuailty {
+    High,
+    Medium,
+    Low,
+}
+fn initialize_pdfium() -> Box<dyn PdfiumLibraryBindings> {
+    let out_path = env::var("OUT_DIR").unwrap_or("".to_string());
+
+    let pdfium_libpath =
+        PathBuf::from(&out_path).join(Pdfium::pdfium_platform_library_name());
+    let bindings = Pdfium::bind_to_library(pdfium_libpath.display())
+        .or_else(|_| Pdfium::bind_to_system_library())
+        .unwrap();
+    bindings
+}
+pub fn render_preview_page(
+    data: &[u8],
+    quailty: PDFQuailty,
+    // font_path: Option<String>,
+) -> DynamicImage {
+    // EVENT_QUEUE.lock().unwrap();
+
+    let render_cfg = PdfBitmapConfig::new();
+    let render_cfg = match quailty {
+        PDFQuailty::High => render_cfg
+            .set_target_width(2000)
+            .set_maximum_height(2000),
+        PDFQuailty::Medium => render_cfg,
+        PDFQuailty::Low => render_cfg.thumbnail(50),
+    }
+    .rotate_if_landscape(PdfBitmapRotation::Degrees90, true);
+    Pdfium::new(initialize_pdfium())
+        .load_pdf_from_bytes(data, None)
```

Comment:
Is it possible to call `initialize_pdfium()` only 1 time and call `load_pdf_from_bytes` multiple times with the same instance of the lib?